app/templates/administrative-units/new.hbs

```diff
@@ -146,6 +146,23 @@
                 </:content>
               </Item>
             {{/if}}
+            {{#if this.isNewIGS}}
+              <Item 
+                @labelFor="geplandeEinddatum" 
+                @required={{true}}
+                @errorMessage={{@model.administrativeUnitChangeset.error.geplandeEinddatum.validation}}
+              >
+                <:label>Geplande einddatum</:label>
+                <:content  as |hasError|>
+                  <AuDatePicker 
+                    @onChange={{fn
+                      (mut @model.administrativeUnitChangeset.geplandeEinddatum)
+                    }}
+                    @error={{hasError}}
+                  />
```

Comment:
I do like this component a lot, but in OP we tend to use `<Datepicker />`. I think that it would be a little weird for consistency not to use the name kind of datepicker accross the application, but might be worth asking Sofie which one she prefers.

If I remember correctly, we had to use a custom component for having a min / max on the dates. We used them when creating new persons or positions, but currently it's Loket which is in charge of creating those resources so we can probably safely change it. Might be worth to make a separate ticket for it though :grin: